brigade-worker/src/k8s.ts

```diff
@@ -461,7 +461,7 @@ export class JobRunner implements jobs.JobRunner {
    */
   public run(): Promise<jobs.Result> {
     return this.start()
-      .then(r => r.wait())
+      .then(r => this.wait())
```

Comment:
Is this a linter recommendation?

```
Promise<JobRunner>.then<jobs.Result, never>(onfulfilled?: (value: jobs.JobRunner) => jobs.Result>
```

Technically, the `r` parameter passed to the promise is a `JobRunner`, so we can safely do `r.wait()` - I'm curious why the change, and if I'm missing something.

Reply:
nah, this was Dimitris-visual-linter :)
probably should have discussed this first, but it seemed to me that both Promises should have either _r.Function_ or _this.Function_. In any case, reverting this as well